README.md

```diff
@@ -22,53 +21,12 @@ def draw():
 
 start_p5(setup, draw)
 ```
+### [Documentation](https://github.com/berinhard/pyp5js/blob/master/docs/index.md)
```

Comment:
Should change the link to point to https://berinhard.github.io/pyp5js/

Reply:
Done!